### DISCUSSION TITLE
visible repositories with private profiles?

### DISCUSSION THREAD

**Discussion by pr-apes** [community/community#20509] [Other Feature Feedback, Questions, & Ideas]
I enabled my profile visibility status to private.

But when forking repositories to submit patches, all repositories are visible to any users.

Would it be possible that this repositories are not visible and accessible from the user profile (or and adjacent tab)?

Many thanks for your help.

**Comment by fahimy143** [comment]
Hi @pr-apes 👋 
instead of enabling your profile status to private just make your repo as a private

**Reply by pr-apes** [reply]
Hi @fahimy143,

forked publish repositories cannot be set to private visibility (due to security reasons).

Many thanks for your help.

**Comment by fahimy143** [comment]
@pr-apes 
You can "hide" the repository from your GitHub profile by deleting the repository from GitHub, but leaving the local repository on your computer (or somewhere else).
You can then later re-add that local repo to GitHub if needed.

**Reply by pr-apes** [reply]
Sorry, @fahimy143, but this is hardly workable.

And even following your instructions, when the repository is added to GitHub, it will be public again.
